gen3-cli/cot/command/query/__init__.py

```diff
@@ -1,70 +1,136 @@
 import os
 import json
+import textwrap
 import click
+from tabulate import tabulate
 from cot.backend.common import exceptions
 from cot.backend import query as query_backend
 from cot.command import root
 
 
-@root.command(
-    'query',
-    context_settings=dict(
-        max_content_width=240
+MAX_TABLE_TEXT_CONTENT_WIDTH = 128
+
+
+@root.group('query')
+def query():
+    pass
+
+
+def blueprint_generation_options(func):
+    options = []
+    options.append(
+        click.option(
+            '-i',
+            '--blueprint-generation-input-source',
+            help='source of input data to use when generating the template'
```

Comment:
for generating I guess, not when. @roleyfoley as a native speaker for better wording